scalafix.conf

```diff
@@ -43,7 +47,6 @@ Disable.symbols = [
     regex = {
       includes = [
         # overrides not fully implemented: https://github.com/scalacenter/scalafix/pull/634
-        "^.*\\Q#asInstanceOf().\\E$"
```

Comment:
Why would you get rid of this?

Reply:
I think it's a mistake, will change.

Reply:
Actually it overlaps `DisableSyntax` rule `noAsInstanceOf` below.

Reply:
:+1: